app/src/main/res/values/strings.xml

```diff
@@ -745,6 +745,7 @@
     <string name="open_privacy_policy">Click here to check our <a href="">privacy policy</a> </string>
     <string name="coordinates_error">Coordinates values are not valid</string>
     <string name="break_glass_error">You cannot enroll TEIs from other org units in this program because it is protected.\nContact your administrator.</string>
+    <string name="break_glass_error_v2">You cannot enroll a %s from other org units in this program because it is protected.\nContact your administrator.</string>
```

Comment:
Same question here why no modify `break_glass_error`?

Reply:
Cause if not languages that already translated the resource won't be notified